src/workflows/Dockerfile

```diff
@@ -3,6 +3,7 @@ RUN pip install --upgrade pip setuptools wheel
 WORKDIR airqo_etl_utils
 COPY airqo_etl_utils/* airqo_etl_utils/
 RUN python airqo_etl_utils/setup.py bdist_wheel
+#TODO: Review packaging process for this microservice
```

Comment:
Thanks! Is this code comment still needed in our codebase? @Mnoble-19

Reply:
Yes @Baalmart , I've tried to make some changes but not yet done. 
package management in python is weird and my knowledge is also still shaky for now thus the TODO, but by end of next sprint I'll probably have read more on it and will remove it

Reply:
Okay @Mnoble-19 , I was told that you are on leave so we shall leave your PR open until you return. I also encourage @BenjaminSsempala to review it asap.